packages/react-server/src/ReactFlightServer.js

```diff
@@ -5534,6 +5536,7 @@ function emitChunk(
   request: Request,
   task: Task,
   value: ReactClientValue,
+  isByteStream: boolean,
```

Comment:
Instead of adding argument in the hottest function call, it would be better to fork this function specifically for this case. Given that you're also just going to assume it's a Uint8Array anyway. In fact, I'd fork `tryStreamTask` too.